doltpy/sql/read.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy.engine import Engine
+import pandas as pd
+from typing import List, Mapping
+from doltpy.shared import rows_to_columns
+
+
+def read_columns(engine: Engine, table: str, as_of: str = None) -> Mapping[str, list]:
+    return read_columns_sql(engine, _get_read_table_asof_query(table, as_of))
+
+
+def read_rows(engine: Engine, table: str, as_of: str = None) -> List[dict]:
+    return read_rows_sql(engine, _get_read_table_asof_query(table, as_of))
+
+
+def read_pandas(engine: Engine, table: str, as_of: str = None) -> pd.DataFrame:
+    return read_pandas_sql(engine, _get_read_table_asof_query(table, as_of))
+
+
+def _get_read_table_asof_query(table: str, as_of: str = None) -> str:
+    base_query = 'SELECT * FROM {table}'.format(table=table)
```

Comment:
nit: you used f-strings in write, also \`table\`

Reply:
Yep, this is a typo, thanks